massa-protocol-exports/src/tests/tools.rs

```diff
@@ -358,12 +346,12 @@ pub async fn assert_banned_nodes(
         tokio::select! {
             msg = network_controller
                    .wait_command(1000.into(), |cmd| match cmd {
-                       NetworkCommand::Ban(node) => Some(node),
+                       NetworkCommand::NodeBanByIds(node) => Some(node),
                        _ => None,
                    })
              =>  {
                  let banned_node = msg.expect("Nodes not banned before timeout.");
-                 nodes.drain_filter(|id| *id == banned_node);
+                 nodes.drain_filter(|id| *id == banned_node[0]);
```

Comment:
Just checking only the first element and not checking the whole list ?

Reply:
Don't hesitate to improve it. I got an issue with this one, there is only one item left, and 4 elements right, so the assertion failed.

Reply:
I was a little bit confused, time based tests are un predictable , especially when debug mode is ON. I'lL try to fix it